ktest.kts

```diff
@@ -45,17 +46,28 @@ assertInPath("gradle")
 
 println("\nTesting ${scriptFile.canonicalPath}...\n")
 """
-    cd $projectLocation
-    gradle clean test
+    gradle -p $projectLocation clean test
 """.trimIndent()
     .execute(
         // Inherit is used so that gradle test output is shown in console to the user
         stdoutRedirectBehavior = ProcessBuilder.Redirect.INHERIT,
         stderrRedirectBehavior = ProcessBuilder.Redirect.INHERIT
     ).let { result ->
-        exitProcess(result.exitCode)
+        if (failOnFailFailure) {
+            exitProcess(result.exitCode)
+        } else {
+            copyTestResultBackToProject()
+        }
     }
 
+fun copyTestResultBackToProject() {
+    """
+        mkdir build
+        mv $projectLocation/build/* build/
+    """.trimIndent()
+        .execute()
+}
```

Comment:
Copying the build folder back over to the original project directory fixed the issue of parsing the test results

https://github.com/mikepenz/action-junit-report/issues/154#issuecomment-841827506